index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="styles1.css" rel="stylesheet">
+        <title>Microverse First Project</title>
+    </head>
+    <body>
+        <header  class="toolbar">
+            <a href="#" class="text1">My Logo</a>
+            <nav>
+                <button class="nav" id="menu" type="button"><img src="Union.png" alt=""></button>
+            </nav>
+        </header>
+        
+        <section class="section">
+            <div class="headline">
+                <h1>I’m Tanmay Glad to see you!</h1>
+                <p> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p class="connect">LET'S CONNECT</p>
+                <ul>
+                    <li><a href="#"><img src="Vector.svg" alt=""></a></li>
+                    <li><a href="#"><img src="Linkedinicon.svg" alt=""></a></li>
+                    <li><a href="#"><img src="Vector(1).svg" alt=""></a></li>
+                    <li><a href="#"><img src="Vector(3).svg" alt=""></a></li>
+                    <li><a href="#"><img src="Vector(2).svg" alt=""></a></li> 
```

Comment:
- [ ] **Required** Let's make the design similar to the figma sample. Your design is missing some icons. Please add them.

| Your design | Figma design |
| --- | --- |
| <img width="309" alt="real2" src="https://user-images.githubusercontent.com/27709832/143614721-29bcf7d3-4472-4edb-949e-244779145bd3.png"> | <img width="284" alt="real" src="https://user-images.githubusercontent.com/27709832/143614517-7dc048d3-88f2-4747-b8a0-500f033e0672.png"> |
| - Compare the font color and correct it | ✅  |
| - Also checkout the icons alignment and match them with the design on the right. | ✅  |

Reply:
sir pleas check the screen shot again. I think you have opened the wrong one.
![Screenshot_2021-11-26-23-12-25-757_com android chrome](https://user-images.githubusercontent.com/92096166/143616432-511c531d-f87a-49dc-9e24-78562acabbe1.jpg)